_episodes/11-skim-milk.md

```diff
@@ -79,7 +79,7 @@ skim_ggH:
 
 Ok, fine. That was way too easy. It seems we have a few issues to deal with.
 
-1. The built code in the `build_skim` job (of the `build` stage) isn't in the `skim_ggH` job by default. We need to use GitLab `artifacts` to copy over this from the right job (and not from `build_skim_latest`).
+1. The code in the `multi_build` jobs (of the `build` stage) isn't in the `skim_ggH` job by default. We need to use GitLab `artifacts` to copy over this from the one of the jobs (let's choose as an example the `multi_build: [rootproject/root:6.28.10-ubuntu22.04]` job).
```

Comment:
`build_skim` is still present in "Set The Right Image" below

Reply:
I see thanks! Will update it